samples/api1.yaml

```diff
@@ -1,14 +1,61 @@
-apiVersion: networking.kuadrant.io/v1beta1
-kind: API
+apiVersion: apps/v1
+kind: Deployment
 metadata:
-  name: dogs-api
+  name: cats
+  labels:
+    svc: cats
 spec:
-  hosts:
-    - "mydogsapi"
-  operations:
-    - name: getdogs
-      path: /dogs
-      method: GET
-    - name: postdog
-      path: /dogs
-      method: POST
+  selector:
+    matchLabels:
+      svc: cats
+  template:
+    metadata:
+      labels:
+        svc: cats
+    spec:
+      containers:
+        - name: cats
+          image: quay.io/3scale/authorino:echo-api
+          env:
+            - name: PORT
+              value: "3000"
+          ports:
+            - containerPort: 3000
+  replicas: 1
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: cats-api
+  annotations:
+    discovery.kuadrant.io/scheme: "http"
+    discovery.kuadrant.io/api-name: "cats"
+    discovery.kuadrant.io/tag: "production"
+    discovery.kuadrant.io/port: "80"
+    discovery.kuadrant.io/oas-configmap: "cats-oas"
+  labels:
+    discovery.kuadrant.io/enabled: "true"
+spec:
+  selector:
+    svc: cats
+  ports:
+    - port: 80
+      protocol: TCP
+      targetPort: 3000
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cats-oas
+data:
+  openapi.yaml: |
```

Comment:
OpenAPI 3.0 is not valid. I spotted two issues:

- required field missing: `info.version`
- required (responses](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.2.md#responsesObject) field missing in the operation object.

Minim valid OAS3:
```
  openapi: "3.0.0"                   
 info:                              
   title: "toy API"                 
   version: "1.0.0"                 
 servers:                           
   - url: http://toys/              
 paths:                             
   /toys:                           
     get:                           
       operationId: "getToys"       
       responses:                   
         200:                       
           description: "success"   
```

consider adding validation call after parsing is done.`